src/typo3/customize.md

```diff
@@ -306,86 +308,105 @@ by this Fluid template:
+        1. Save and return
+    * Admin Tools::Extensions:
+        1. Click on the update icon of the Aimeos extension
 
-Some plugins display e.g. the category tree of the shop/site to be able to select one of those categories. To display the available options from the required shop/site, please insert an additional line into the PageTS field of the page:
+    The "update" function runs through a list of checks. Once this check has finished executing, you'll be presented with the types of data that have been added to the new site.
 
-![PageTS for multiple sites](Aimeos-pagets.png)
+    !!! warning
+        If you update Aimeos from a previous version, you need to run the update script for all sites you've created! Otherwise, required records may be missing and existing data isn't migrated.
 
-* Web::Page
-    1. Select second page tree
-    1. Edit page
-    1. Tab *Resources*, section *TypoScript Configuration*
+3. **Create the new "Aimeos locale"**
 
-```typoscript
-tx_aimeos.mshop.locale.site = <code of site>
-```
+    In the Aimeos administraion interface of the TYPO3 backend, open the "Locales" menu and click on "Sites". Add a new site and enter the new "site code" in the required "Code" field.
 
-Click on the icon for *Save and close document* at the top to store the change. Repeat these steps for each page tree where an other shop site other than the "default" site should be used. Clear all TYPO3 caches when you are done.
+4. **Add a pageTS config**
 
-## TypoScript config
+    Back in TYPO3's page tree, go to the root page of the newly created shop/site. In order to display the appropriate data of the respective (second, third,...) shop/site, add an additional setting to the page's pageTS field:
 
-To tell the front-end that it should use another shop/site by default instead of the standard one ("default"), add a line of TypoScript to the setup config:
+    ![PageTS for multiple sites](Aimeos-pagets.png)
 
-![Aimeos setup TS configuration](Aimeos-setupts.png)
+    * Web::Page
+        1. Select second page tree
+        2. Edit page
+        3. Tab *Resources*, section *TypoScript Configuration*
 
-* Web::Template
-    1. Select second page tree
-    1. Choose *Info/Modify* in the drop-down at the top
-    1. Click on *Setup* in the box below
+    ```typoscript
+    tx_aimeos.mshop.locale.site = <code of site>
+    ```
 
-```typoscript
-plugin.tx_aimeos.settings.mshop.locale.site = <code of site>
-```
+    Click on the icon for *Save and close document* at the top to store the change. Repeat these steps for each page tree where an different shop site other than the "default" site should be used. Clear all TYPO3 caches when you are done.
 
-Click on the icon for *Save and close document* at the top to store the change. Repeat these steps for each page tree where an other shop site other than the "default" site should be used. Clear all TYPO3 caches when you are done.
+5. **TypoScript config**
```

Comment:
Would be better to use headlines (## TypoScript config) instead ordered lists with only one element

Reply:
This is the 5th in a list of 6 (or 7?) elements. (The texts below a list element are indented.)